node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -85,6 +87,10 @@ impl ChainScraper {
 	/// As long as we have `MAX_FINALITY_LAG` this makes sense as a value.
 	pub(crate) const ANCESTRY_SIZE_LIMIT: u32 = MAX_FINALITY_LAG;
 
+	/// How many blocks after finalization a backed and not included candidate
+	/// should be kept.
+	pub(crate) const CANDIDATE_LIFETIME_AFTER_FINALIZATION: BlockNumber = 2;
```

Comment:
Some argument why we are doing this at all (abandoned forks, ensuring slashes to happen as much as possible, ...) would be good.

Reply:
Done